src/main/java/io/mosip/mimoto/dto/openid/presentation/FieldDTO.java

```diff
@@ -1,16 +1,16 @@
 package io.mosip.mimoto.dto.openid.presentation;
 
+
 import lombok.AllArgsConstructor;
 import lombok.Builder;
 import lombok.Data;
 import lombok.NoArgsConstructor;
 
-import java.util.List;
-
 @Data
+@Builder
 @NoArgsConstructor
 @AllArgsConstructor
-@Builder
-public class LDPVc {
-    List<String> proofTypes;
+public class FieldDTO {
+    String[] path;
+    FilterDTO filter;
```

Comment:
As per our initial discussion, we thought we will do the minimal set for the pilot release.

Since Mimoto is the service geenrating the presenationDefinition and it is the service who is consuming it for now. we don't expect additional fields also.

Hence went ahead will the minimal fields

Reply:
ok for now